README.md

```diff
@@ -76,9 +76,15 @@ Parasoft Demo Application exposes port 9001 for the user to connect to the HSQLD
 | Username | `SA`                                                |
 | Password | `pass`                                              |
 
-## Using Parasoft JMS Proxy and Virtual Asset
+## Using Parasoft JMS Proxy and Virtual Asset with message queue
```

Comment:
I think in general we can simplify this whole section to prevent duplicate text. So I put both mq type under this same section.